fastly/fastly.go

```diff
@@ -173,24 +183,48 @@ func fatalIfErr(log *logrus.Logger, err error) {
 	}
 }
 
+func writeTimestamp(log *logrus.Logger, config Config, timestamp int) {
+	err := ioutil.WriteFile(config.TimestampFileLocation, []byte(fmt.Sprintf("%d", timestamp)), 0644)
+	if err != nil {
+		log.WithFields(logrus.Fields{
+			"error": err.Error(),
+		}).Error("error writing fastly timestamp file")
+	}
+}
+
+func readTimestamp(log *logrus.Logger, config Config) string {
+	defaultResult := "h"
```

Comment:
I think this should be `0`

>For the first request, the timestamp can be 0. The response will then return a new timestamp field which should be used for the next request.

https://docs.fastly.com/api/analytics#url_structure

Reply:
They both return the next timestamp to use, so my thought on using h instead is that when we start it up initially, it will first pull the last 120 seconds. Then the next request will pick up the timestamp from that first request, and use that for the remaining life of the container. Also, from doing some testing, it looks like even if you use 0, it goes back 120 seconds, so while it doesn't say either way in the docs, I believe it is more or less the same.

Reply:
Cool, works for me